images/fcos-bastion-image/root/usr/bin/hpecmd

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+if [ "${#}" -lt 1 ]; then
+    echo "Usage: ${0} <host_id> <command>"
+    exit 1
+fi
+
+host_id="${1}"
+shift
+
+host_obj=$(sed '1s/^#//; 2,${/^#/d; /^$/d}' "/etc/hosts_pool_inventory" | yq -p csv '.[] | select(.bmc_address == "*'".${host_id}"'*")')
+
+user=$(echo "${host_obj}" | yq -r '.bmc_user')
+password=$(echo "${host_obj}" | yq -r '.bmc_pass')
+address=$(echo "${host_obj}" | yq -r '.bmc_address')
+echo -n "${address}: "
+p=$(mktemp)
+echo "${password}" > $p
+sshpass -f $p ssh -o ServerAliveInterval=30 -o ServerAliveCountMax=1000 "${user}"@"${address}" "${@}"
+rm -f $p
```

Comment:
Can we do this via trap to ensure that the removal happen even when sigint and other typical signals block the execution flow?

Reply:
trapped